src/inspect.rs

```diff
@@ -457,16 +499,16 @@ pub fn handle_inspect_inner(inspect: &Inspect) -> Result<InspectionResults> {
                             )
                         })
                         .into(),
-                    iterations: authorizer_builder.iterations(),
-                    elapsed: authorizer_builder.execution_time(),
+                    iterations: authorizer.iterations(),
+                    elapsed_micros: authorizer.execution_time().unwrap_or_default().as_micros(), // TODO handle failed evaluations?
```

Comment:
this is a bit tricky to handle correctly: the field can indeed be empty if datalog fact generation fails.
If datalog fact generation succeeds, but authorization fails because of a datalog error in the policy, it will be defined.
If datalog fact generation succeeds, but authorization fails without a datalog error in the policy (ie no allow policy matched or a deny policy matched, it will also be defined).

So i think `elapsed_time` should be an option here, same as in `Authorizer`, but then that makes display a bit more awkward